spec/services/okta_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+require "json/jwt"
+
+RSpec.describe Okta do
+  let(:okta_user_info) do
+    {
+      sub: "qwer",
+      email: "okta@okta.com",
+      first_name: "Okta",
+      lsat_name: "Test"
+    }
+  end
+  let(:jwt_payload) { okta_user_info.merge(exp: 1.minute.from_now.to_i) }
+  let(:jwt_default_payload) { {iss: ENV["OKTA_SERVER_PATH"], aud: ENV["OKTA_SERVER_AUDIENCE"]} }
+  let(:rsa_private) { OpenSSL::PKey::RSA.generate 2048 }
```

Comment:
What is the `rsa_private`? I see its used to create the `jwk`.

Reply:
RSA is a type of encryption key. Here we create a new public/private key for each test and then test decoding with that key.